.setup/bin/sample_courses/models/course/course_create.py

```diff
@@ -161,25 +160,20 @@ def create(self):
         self.electronic_gradeable_version = Table("electronic_gradeable_version", self.metadata, autoload=True)
         for gradeable in self.gradeables:
             gradeable.create(self.conn, self.gradeable_table, self.electronic_table, self.peer_assign, reg_table, self.component_table, self.mark_table)
-            form = os.path.join(self.course_path, "config", "form", "form_{}.json".format(gradeable.id))
+            form = os.path.join(self.course_path, "config", "form", f"form_{gradeable.id}.json")
             with open(form, "w") as open_file:
                 json.dump(gradeable.create_form(), open_file, indent=2)
-        os.system("chown -f submitty_php:{}_tas_www {}".format(self.code, os.path.join(self.course_path, "config", "form", "*")))
+        os.system(f"chown -f submitty_php:{self.code}_tas_www {os.path.join(self.course_path, 'config', 'form', '*')}")
         if not os.path.isfile(os.path.join(self.course_path, "ASSIGNMENTS.txt")):
-            os.system("touch {}".format(os.path.join(self.course_path, "ASSIGNMENTS.txt")))
-            os.system("chown {}:{}_tas_www {}".format(self.instructor.id, self.code,
-                                                      os.path.join(self.course_path, "ASSIGNMENTS.txt")))
-            os.system("chmod -R g+w {}".format(self.course_path))
-            os.system("su {} -c '{}'".format("submitty_daemon", os.path.join(self.course_path,
-                                                                          "BUILD_{}.sh".format(self.code))))
-            #os.system("su {} -c '{}'".format(self.instructor.id, os.path.join(self.course_path,
-            #                                                              "BUILD_{}.sh".format(self.code))))
```

Comment:
Unrelated to this PR: What was the purpose of this line originally?  Is this something obsolete, or something we should re-enable eventually?

Reply:
It looks like it changes the user of the system to be instructor, but I'm not too sure about the purpose of it

Reply:
Perhaps you can ask Barb what the expected owner of this directory is

Reply:
https://github.com/Submitty/Submitty/commits/3aab0701aeec42b2aa03763009e372d037a89849 provides some context, where the top commit adds this commented out line. Looking at the history, we were requiring running `BUILD_{self.code}.sh` as an instructor user originally, but we migrated away from that as we introduced the various `submitty_*` users. This line was re-added as part of a debug of a bug around these new users that Barb was trying to quickly fix, but in the end wasn't necessary, and so removing it is fine, and has been commented out for the entire 5 years of its existence